src/window/window_wrapper/mod.rs

```diff
@@ -52,7 +52,9 @@ pub struct GlutinWindowWrapper {
     mouse_manager: MouseManager,
     title: String,
     fullscreen: bool,
+    first_render_pass: bool,
```

Comment:
would it make sense to combine first_render_pass and saved_grid_size into a single option which starts out as None and changes to Some((u64, u64)) after a render has been made? I didn't fully walk through the logic here so I may be missing something.

Reply:
Well, i changed it and it works. But i don't know if it made code better or worse :)

I think it's good enough :)